packages/neos-ui-redux-store/src/CR/Workspaces/index.js

```diff
@@ -9,13 +9,17 @@ import * as selectors from './selectors';
 
 const UPDATE = '@neos/neos-ui/CR/Workspaces/UPDATE';
 const PUBLISH = '@neos/neos-ui/CR/Workspaces/PUBLISH';
-const DISCARD = '@neos/neos-ui/CR/Workspaces/DISCARD';
+const COMMENCE_DISCARD = '@neos/neos-ui/CR/Workspaces/COMMENCE_DISCARD';
```

Comment:
Nitpick or maybe by french knowledge ... Commence is valid english, but unusual in this context why not "BEGIN_DISCARD" ...

Reply:
Totally agree, but for the sake of consistency with other sagas where we use commence, e.g. delete node saga.